charts/velero/crds/backups.yaml

```diff
@@ -1,11 +1,15 @@
+
```

Comment:
@jenting this looks good. Please rebase when you have a chance so we can check again.

I was going to ask to remove the first empty line, but I noticed that our generated CRDs are like that. Weird. Don't worry about it then.

Reply:
Rebase, please check again 🙏